pkg/finder/fsfinder.go

```diff
@@ -135,10 +158,31 @@ func (fsf FileSystemFinder) findOne(pathRoot string) ([]FileMetadata, error) {
 
 				walkFileExtension := strings.TrimPrefix(filepath.Ext(path), ".")
 
+				// Checking for file name matching with additional files
+				// Check matching base name, return vals with passed KVPair value as fileType
+				//if _, ok := fsf.AdditionalFiles[dirEntry.Name()]; ok {
+				//	extensionLowerCase := AdditionalFiles[dirEntry.Name()]
+				//}
+
+				if len(walkFileExtension) == 0 {
+					var ok bool
+					walkFileExtension, ok = fsf.AdditionalFiles[dirEntry.Name()]
+					if !ok {
+						walkFileExtension = ""
+					}
```

Comment:
```suggestion
					if walkFileExtension, ok := fsf.AdditionalFiles[dirEntry.Name()]; ok {
						walkFileExtension = ""
					}
```

Reply:
Unfortunately I tried this, the := conditional shadows the outer variable and declares a new instance of walkFile instead. Found a more concise workaround recently though, so I hope to update the MR with it soon.